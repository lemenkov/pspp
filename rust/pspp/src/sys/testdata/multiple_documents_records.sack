# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 2; 0; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Numeric variables, no label or missing values.
2; 0; 0; 0; 0x050800 *2; s8 "NUM1";
2; 0; 0; 0; 0x050800 *2; s8 "NUM2";

# Two document records.
(6; 1; s80 "One line of documents") >>* 2<<;

# Character encoding record.
7; 20; 1; 12; "windows-1252";

# Dictionary termination record.
999; 0;

# Data.
1.0; 2.0;