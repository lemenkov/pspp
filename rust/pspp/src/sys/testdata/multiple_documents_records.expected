╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │None                    │
│       Number of Cases│Unknown                 │
╰──────────────────────┴────────────────────────╯

╭─────────┬─────────────────────╮
│Variables│                    2│
│Documents│One line of documents│
│         │One line of documents│
╰─────────┴─────────────────────╯

                                               Variables
╭────┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│    │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├────┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│num1│       1│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
│num2│       2│     │                 │Input│    8│Right    │F8.0        │F8.0        │              │
╰────┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯

╭────┬────┬────╮
│Case│num1│num2│
├────┼────┼────┤
│1   │1.00│2.00│
╰────┴────┴────╯
